wiegand.py

```diff
@@ -8,6 +8,7 @@
 import utime
 
 CARD_MASK = 0b11111111111111110 # 16 ones
+FACILITY_MASK = 0b11111111 # 8 ones
```

Comment:
I think this should be 
`0b1111111100000000000000000`
so that (new) line 61 becomes:
`
    return ( self.last_card & FACILITY_MASK ) >> 17
`
to be consistent with the CARD_MASK usage.  That is, it should be the mask _before_ shifting, not after.

Reply:
Good call. I've also tested this with my 26bit card and it works just as well.

Reply:
Okay, expect v1.1.0 to appear momentarily.  Thanks for the contribution!